PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -120,9 +122,9 @@ struct femtoDreamProducerTask {
   Configurable<std::vector<float>> ConfV0DaughPIDnSigmaMax{"ConfV0DaughPIDnSigmaMax", std::vector<float>{5.f, 4.f}, "V0 Daugh sel: Max. PID nSigma TPC"};
 
   /// \todo should we add filter on min value pT/eta of V0 and daughters?
-  Filter v0Filter = (nabs(aod::v0data::x) < V0DecVtxMax.value) &&
+  /*Filter v0Filter = (nabs(aod::v0data::x) < V0DecVtxMax.value) &&
```

Comment:
see above

Reply:
I commented it because Valentina is not currently using it, it is not related to the trigger.